src/crypto/index.js

```diff
@@ -266,17 +266,18 @@ export const validateMnemonic = bip39.validateMnemonic
 /**
  * Get a private key from mnemonic words.
  * @param {string} mnemonic the mnemonic phrase words
+ * @param {int} index the address index
  * @param {Boolean} derive derive a private key using the default HD path (default: true)
  * @return {string} hexstring
  */
-export const getPrivateKeyFromMnemonic = (mnemonic, derive = true) => {
+export const getPrivateKeyFromMnemonic = (mnemonic, index = 0, derive = true) => {
```

Comment:
Would it be possible to move `index` to become the last arg so as to not break existing usages of this method?

Reply:
Good call.  Pushed.